application/src/main/java/com/musicforall/web/FileController.java

```diff
@@ -0,0 +1,59 @@
+package com.musicforall.web;
+
+import com.musicforall.files.FileManager;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.annotation.PostConstruct;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * @author Evgeniy on 11.06.2016.
+ */
+
+@Controller
+public class FileController {
```

Comment:
- [ ]  make more like rest api files/ POST to upload, files/{fileName} GET to retrieve file
- [ ]  all file-specific logic should be moved to file manager (directory creation, exception handling, etc)